changelog/fragments/3040-fix-loglevel.yaml

```
@@ -0,0 +1,6 @@
+entries:
+  - description: >
+      Add "panic" level for --zap-stacktrace-level (allows "debug", "info", "error", "panic")
+    kind: "bugfix"
```

Nit: This is probably more of a new feature than a bug fix. Can you change this to `addition`?
```suggestion
    kind: "addition"
```